runtime/j9vm/jvm.c

```diff
@@ -1717,12 +1717,56 @@ JNI_CreateJavaVM_impl(JavaVM **pvm, void **penv, void *vm_args, BOOLEAN isJITSer
 	 * trying to load application native libraries that are linked against
 	 * libraries in /usr/lib we could fail to find those libraries if /usr/lib
 	 * is not on the LIBPATH.
-	 * */
-	addToLibpath("/usr/lib", FALSE);
```

Comment:
I think a better solution would be to fix `addToLibpath` so it doesn't add entries that are already present, or perhaps better, eliminates all redundancy. Other entries are added to `LIBPATH` that might already be present (e.g. a VM process launching a child will end up with two (or more) occurrences `.../lib/j9vm` and `.../lib/ppc64/compressedrefs`).

Reply:
`addToLibpath` has two modes: prepend and append.  We can remove the duplication in append mode but not prepend.

Considering this is the only append location, I'd rather handle it here then make the general function more complex

Reply:
We should remove the duplicate even in the prepend case: it's never helpful to have an entry listed twice. I'm fine with deferring the more general improvement.

Reply:
I don't plan to do that in this PR.  Resolving this conversation and deferring that action